files/default/neutron-ha-tool.py

```diff
@@ -249,13 +264,48 @@ def run(args):
 
 
 class RandomAgentPicker(object):
-    def __init__(self, agents):
+    def __init__(self, qclient, agents):
         self.agents = agents
 
     def pick(self):
         return random.choice(self.agents)
 
 
+class LeastBusyAgentPicker(object):
+    def __init__(self, qclient, agents):
+        self.now = datetime.datetime.now
+        self.cache_created_at = None
+        self.qclient = qclient
+        self.agents_by_id = dict((agent['id'], agent) for agent in agents)
+        self.router_count_per_agent_id = dict()
```

Comment:
you can ignore this, but I would use `{}`, `[]` or `()` to initialize empty dicts, lists or tuples, the reason is that is marginally faster and more usual